sql/executor_test.go

```diff
@@ -26,6 +26,7 @@ func goodStream(stream chan interface{}) (bool, string) {
 	keepSize := 10
 
 	for rsp := range stream {
+		log.Debug(rsp)
```

Comment:
delete the debug code ?

Reply:
I find myself adding and deleting this line every time I try to figure out the bug in CI. So I think it is helpful to keep it.

In deployment, this line will be skipped, since we only do `log.Error` and `log.Warning`. :)